helm/sematic-server/values.yaml

```diff
@@ -1,32 +1,33 @@
 auth:
   enabled: true
-  google_oauth_client_id: 977722105393-257kdkrc5dfbpu0jcsd8etn1k4u4q4ut.apps.googleusercontent.com
-  authorized_email_domain: sematic.ai
+  google_oauth_client_id: xxxxxxxx.apps.googleusercontent.com # REPLACE ME
+  authorized_email_domain: example.org # REPLACE ME
+
 aws:
   enabled: true
-  storage_bucket: sematic-dev
+  storage_bucket: my-s3-bucket # REPLACE ME
+
+secret:
+  create: true
 
-create_secret: true
 database:
-  url: postgres://postgresql:5432/postgres
+  url: postgres://postgresql:5432/postgres # REPLACE ME
 
 replicaCount: 1
 
 image:
-  repository: sematicai/sematic-server
+  #repository: sematicai/sematic-server:v0.20.0
   pullPolicy: IfNotPresent
-  tag: v0.15.1
+  #pullSecrets: []
```

Comment:
Is the comment intentional?

Reply:
Yes.  No image pull secrets are needed to install our default chart since the `sematicai/sematic-server` is a public image that requires no credentials to pull.